sentry_sdk/integrations/opentelemetry/potel_span_processor.py

```diff
@@ -59,11 +59,12 @@ def on_end(self, span):
         if is_sentry_span(span):
             return
 
-        if span.parent and not span.parent.is_remote:
-            self._children_spans[span.parent.span_id].append(span)
-        else:
+        is_root_span = not (span.parent and not span.parent.is_remote)
```

Comment:
```suggestion
        is_root_span = not span.parent or span.parent.is_remote
```

Reply:
its the same thing, but I can update (rule of de morgan or however it is called in english)